### ISSUE TITLE
cer greater than 1

### ISSUE THREAD

**Issue by wumiaoshu** [FactoDeepLearning/DAN#24]
Hi, I am reproducing your results. As per your instructions, I first pre-trained a line recognition model and then trained it on the Read2016 single-page dataset. I have been training for three days, and I noticed that the CER on the validation set remains above 1. Could you please tell me what might be the reason for this?

<img width="838" height="121" alt="Image" src="https://github.com/user-attachments/assets/81ecb139-f3cf-46ae-b37f-2bf6f9167e1c" />

**Comment by FactoDeepLearning** [comment]
Hi, it suggests that the model performs very badly, and that it predicts many more characters than there are on the image, leading to CER > 100%. This can be explained by the lack of training data (if you do not use synthetic samples for instance), or training data not similar enough compared to validation data (curriculum synthetic data seems not to be ended in your example).
